requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
I have resolved dependencies using python3.8
As the upcoming change in [GitHub actions PR](https://github.com/edx/dbt-schema-builder/pull/50) will be running python3.8 CI tests for this repository